app/controllers/ish_manager/venues_controller.rb

```diff
@@ -14,34 +14,45 @@ def index
 
   def new
     @venue = Venue.new
+    @venue.city_id = params[:city_id] if params[:city_id]
+    @venue.to_city = 'yes' if params[:city_id]
+
     authorize! :new, @venue
   end
 
   def create
+    redirect_path = venues_path()
```

Comment:
parens are optional, you should not have parent `()` unless you're chaining methods and the parens resolve ambiguity.